backend/src/main/java/com/mcmasterbaja/analyzer/AccelCurveAnalyzer.java

```diff
@@ -45,139 +40,18 @@ public void analyze() {
             Arrays.copyOfRange(outputFiles, 0, 2),
             new String[] {"Timestamp (ms)", inputColumns[0], inputColumns[1]},
             Arrays.copyOfRange(outputFiles, 2, 3));
     linearInterpolate.analyze();
-
-    // LinearInterpolaterAnalyzer linearInt = new
-    // LinearInterpolaterAnalyzer(Arrays.copyOfRange(outputFiles, 0, 2),
-    // Arrays.copyOfRange(outputFiles, 2, 3));
-    // linearInt.analyze();
-
-    // Here is all the analyzing done for combining the files, the rest concerns
-    // finding individual runs
-
-    // This all was to also write the data to a separate file, but it had the same data as the
-    // original file, so unnecessary for now
-    // String path = System.getProperty("user.dir");
-    // String output = path + "/upload-dir/accelCurve.csv"; // This is bad for concurrency
-
-    Reader reader = new CSVReader(outputFiles[2]);
-    List<List<String>> dataPoints = reader.read();
-
-    // Writer writer = new CSVWriter(outputFiles[2]);
-    // writer.write(dataPoints);
-
-    List<List<Integer>> accelTimes = getAccelTimestamp(dataPoints);
-    for (int i = 0; i < accelTimes.size(); i++) {
-      System.out.println(accelTimes.get(i));
-    }
-    try {
-      timestampToCSV(accelTimes, dataPoints);
-    } catch (IOException e) {
-      e.printStackTrace();
-    }
-
-    String[] files = new String[accelTimes.size()];
-    for (int i = 0; i < accelTimes.size(); i++) {
-      files[i] = "./data/run" + i + ".csv";
-    }
-  }
-
-  // Outdated
-  // Currently it uses a sliding window + interpolation to get the dataRPM, and
-  // further uses a rolling average on the longer file
-  public List<List<String>> InterpolateAndRollAverage(
-      List<List<String>> longerFile, List<List<String>> shorterFile) {
-
-    List<List<String>> dataPoints = new ArrayList<List<String>>();
-    List<String> dataPoint = new ArrayList<String>(3);
-
-    // Add the first data point
-    dataPoint = new ArrayList<String>(longerFile.get(0));
-    dataPoint.add(shorterFile.get(0).get(1));
-    dataPoints.add(dataPoint);
-
-    // Reset
-    dataPoint = new ArrayList<String>(3);
-
-    // Average the first 30 lines of the longer file to get the initial RPM
-    double rollSum = 0;
-
-    int indexShort = 2;
-    // Begin loop
-    int limit = longerFile.size();
-    for (int i = 2; i < limit; i++) {
-      // Find time in longer file
-      int timeLong = Integer.parseInt(longerFile.get(i).get(0));
-
-      // Find time in shorter file
-      int timeShort = Integer.parseInt(shorterFile.get(indexShort).get(0));
-
-      // Find the closest time in the shorter file
-      while (timeLong >= timeShort) {
-        indexShort++;
-        // Catch out of bounds
-        if (indexShort >= shorterFile.size()) {
-          indexShort--;
-          break;
-        }
-        timeShort = Integer.parseInt(shorterFile.get(indexShort).get(0));
-      } // Keep going until you have a time that is greater than the time in the longer
```

Comment:
Was this just another version of the interpolate? Is this why its all gone now since u put the pro one in.

Reply:
Yeah old code from before rolling average and interpolate were separate. Hasn't been used in forever